5.5/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:centos7
+
+ADD mysql /opt/openshift/
+ENV PATH /opt/openshift/bin:$PATH
+
+WORKDIR /opt/openshift
+RUN create_user.sh
```

Comment:
@mnagy it's not about the size, it's about the clutter.  it also makes it harder to read the Dockerfile because i need to go jump off to some other script to know what you're doing in it.  I've never seen a Docker image that does this, can you show me one?